docs/clients/logstash/README.md

```diff
@@ -0,0 +1,98 @@
+# Logstash
```

Comment:
This is awesome ! Can you include also an example of a config filled in with configuration at the bottom?.

Reply:
Sure